src/main/resources/bidder-config/ix.yaml

```diff
@@ -1,5 +1,6 @@
 adapters:
   ix:
+    # Please contact the maintainer to obtain endpoint details
```

Comment:
To confirm - Index received email from Prebid Info<info@prebid.org> for this PR. Notification is working effectively! 🎉

Reply:
Question: are we doing same for PBS-Go?

Reply:
> are we doing same for PBS-Go?

Now that we've tested it in this repo, I'll open a PR to get the same behavior in the PBS-Go repo.